flexible/mobile-backend/pom.xml

```diff
@@ -0,0 +1,98 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud.solutions.flexenv</groupId>
+  <artifactId>backend</artifactId>
+  <packaging>war</packaging>
+  <version>0.0.1-SNAPSHOT</version>
+  <name>App Engine Backend module for Firebase</name>
+  <url>https://cloud.google.com</url>
+
+  <!--
+    The parent pom defines common style checks and testing strategies for our samples.
+    Removing or replacing it should not affect the execution of the samples in anyway.
+  -->
+  <parent>
+    <groupId>com.google.cloud.samples</groupId>
+    <artifactId>shared-configuration</artifactId>
+    <version>1.0.11</version>
+  </parent>
+
+  <properties>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>1.8</maven.compiler.source>
+
+    <appengine.target.version>1.9.30</appengine.target.version>
```

Comment:
That is really old

Reply:
While this works and will get updated by DPEBot, we moved to putting the versions explicitly in the dependencies as it's easier for the user to figure out.

Reply:
It probably shouldn't even be used on Flex as compat is deprecated if I recall.